packages/desktop-client/src/components/select/RecurringSchedulePicker.jsx

```diff
@@ -48,19 +48,18 @@ function parsePatternValue(value) {
 }
 
 function parseConfig(config) {
-  return (
-    config || {
-      start: monthUtils.currentDay(),
-      interval: 1,
-      frequency: 'monthly',
-      patterns: [createMonthlyRecurrence(monthUtils.currentDay())],
-      skipWeekend: false,
-      weekendSolveMode: 'before',
-      endMode: 'never',
-      endOccurrences: '1',
-      endDate: monthUtils.currentDay(),
-    }
-  );
+  return {
+    start: monthUtils.currentDay(),
+    interval: 1,
+    frequency: 'monthly',
+    patterns: [createMonthlyRecurrence(monthUtils.currentDay())],
+    skipWeekend: false,
+    weekendSolveMode: 'before',
+    endMode: 'never',
+    endOccurrences: '1',
+    endDate: monthUtils.currentDay(),
+    ...config,
```

Comment:
Patching another issue while I'm working here.

Before: if `config` object was incomplete - some fields in the form would be blank.

After: the fields are always filled in with default values.